### DISCUSSION TITLE
I can't use the PS5 gamepad in Genshin Impact in Play Cover 3.0.0 and 3.0.1. The game crashes immediately when you press any button on the gamepad.

### DISCUSSION THREAD

**Discussion by kkurrill** [PlayCover/PlayCover#1032] [Q&A] [answered]
On earlier versions of PlayCover, it is not possible to log in to this game. Versions 3.0.0 and newer work fine, except for trying to use the gamepad. I tried reinstalling Command Tools several times - it didn't help. and also on earlier versions 2.0.3 and older, the game starts completely incorrectly, but the gamepad does not turn off the game in any way. I have a desire to play only with a gamepad. Please tell me what I can do.

**Comment by kkurrill** [comment]
[report.docx](https://github.com/PlayCover/PlayCover/files/11966996/report.docx)

**Reply by Sehyn** [reply]
Im currently using a PS5 Controller nothing goes wrong for me check perhaps OS version? I'm using Sonoma 14.

**Comment by MEMORIEmusic** [comment, marked as answer]
Right click the game -> Disable Keymapping and Smart Keymapping

[on PlayCover 3.0]
